.github/workflows/ci.yaml

```diff
@@ -17,21 +17,3 @@ jobs:
         run: python3 -m pip install tox
       - name: Run linters
         run: tox -e lint
-
```

Comment:
ugh I am a noob, why is this removed?

Reply:
as for vm, there's no tests in place - this would just fail CI now